Cargo.toml

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2024 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# SPDX-License-Identifier: Apache-2.0
+
+[workspace]
+resolver = "2"
+members = [
+    "up-streamer",
+    "examples/simple-example",
+]
+
+[workspace.package]
+rust-version = "1.66.1"
```

Comment:
Via @sophokles73:
> is this aligned (and verified) with all of your upstream dependencies?

Reply:
Via @PLeVasseur:
> Can check.
> 
> What's the best way to do this? Checking dependencies' Cargo.toml manually?

Reply:
ping @sophokles73

Reply:
I guess the best way to do this would be to try to compile with a rust compiler of that version.

Reply:
Think I'll bump this to 1.76.0 based on our discussions [over here](https://github.com/eclipse-uprotocol/up-rust/issues/71).